docs/products/kafka/howto/flink-with-aiven-for-kafka.rst

```diff
@@ -0,0 +1,200 @@
+Use Apache Flink® with Aiven for Apache Kafka®
+======================================================
+
+`Apache Flink® <https://flink.apache.org/>`_ is an open source platform for processing distributed streaming and batch data.
+Where Apache Kafka® excels at receiving and sending event streams, Flink consumes, transforms, aggregates, and enriches your data.
```

Comment:
I would add a paragraph after this saying that you can experience the power of streaming SQL transformation using Aiven for Apache Flink (with the link to the relevant docs sections).

Reply:
My clarification (now below) used to be right here!  I'll move it back up to the top and use your wording.

I'll ask someone to follow along and check the code.